docs/1.10/administration/configure-authentication/oidc-auth.md

```diff
@@ -5,26 +5,29 @@ weight: 25
 
 ### Configure Your OIDC Provider
 
 You must configure your OIDC provider so that you can use it with Harbor. For precise information about how to perform these configurations, see the documentation for your OIDC provider.
 
-- Set up the users and groups that will use the OIDC provider to log in to Harbor. You do not need to assign any specific OIDC roles to users or groups as these do not get mapped to Harbor roles.
-- The URL of the OIDC provider endpoint, known as the Authorization Server in OAuth terminology, must service the well-known URI for its configuration document. For more information about the configuration document, see the [OpenID documentation](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest).
-- To manage users by using OIDC groups, create a custom group claim that contains all of the user groups that you want to register in Harbor. The group claim must be mapped in the ID token that is sent to Harbor when users log in. You can enable the `memberof` feature on the OIDC provider. With the `memberof` feature, the OIDC user entity's `memberof` attribute is updated when the group entity's `member` attribute is updated, for example by adding or removing an OIDC user from the OIDC group.
+- Set up the users and groups which will use the OIDC provider to log in to Harbor. You do not need to assign any specific OIDC roles to users or groups as these do not get mapped to Harbor roles.
+- The URL of the OIDC provider endpoint, known as the Authorization Server in OAuth terminology, must serve the well-known URI for its configuration document. For more information about the configuration document, see the [OpenID documentation](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest).
+- To manage users by using OIDC groups, create a custom group claim which contains all of the user groups that you want to register in Harbor. The group claim must be mapped in the ID token which is sent to Harbor when users log in. You can enable the `memberof` feature on the OIDC provider. With the `memberof` feature, the OIDC user entity's `memberof` attribute is updated when the group entity's `member` attribute is updated, for example by adding or removing an OIDC user from the OIDC group.
 - Register Harbor as a client application with the OIDC provider. Associate Harbor's callback URI to the client application as a `redirectURI`. This is the address to which the OIDC provider sends ID tokens.
 
 ### Configure an OIDC Provider in Harbor
 
 Before configuring an OIDC provider in Harbor, make sure that your provider is configured correctly according to the preceding section.
 
-1. Log in to the Harbor interface with an account that has Harbor system administrator privileges.
+1. Log in to the Harbor interface as an account with Harbor system administrator privilege.
```

Comment:
Log in to the Harbor interface with an account that has Harbor system administrator privileges.